Module 5/Ex0ToEx5 V2/Sources/Intern.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Intern.hpp"
+#include "PresidentialPardonForm.hpp"
+#include "RobotomyRequestForm.hpp"
+#include "ShrubberyCreationForm.hpp"
+#include "memory"
+
+std::unique_ptr<Form> Intern::makeForm(const std::string &formName, const std::string &target)
+{
+  std::map<std::string, FormCreator> formCreators = {

```

Comment:
This map is not supposed to be modified. Make it constant.
Although not what is specified in the exercise, use an enumeration instead of a string.
This is preferable to have better control over the possible values.